src/status_im2/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -68,7 +71,7 @@
            :index             index
            :header-text       header-text
            :header-background header-background}
-          (when background background)])]]
+          background])]]
```

Comment:
changed `(when background background) => background`

since unless `background` is `false` (which is not because we expect a component/vector), it is the same